azimuth/modules/dataset_analysis/dataset_warnings.py

```diff
@@ -278,11 +278,10 @@ def get_syntactic_warnings(
 
         return [
             DatasetWarning(
-                name=f"Length mismatch "
-                f"(>{max_delta_mean_tokens}±{max_delta_std_tokens} tokens)",
-                description=f"Delta between the number of tokens of a "
+                name=f"Length mismatch " f"(>{max_delta_mean_words}±{max_delta_std_words} words)",
+                description=f"Delta between the number of words of a "
```

Comment:
```suggestion
                description=f"Delta between the number of words per sample for a "
```
or "per utterance" or "per example," if not "per sample?" I realize that phrasing isn't new, but if you don't mind updating, I think this would be more clear!

Reply:
I picked utterance because I don't think we use 'sample' often in the UI.